spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -742,16 +743,20 @@ private static String sqlString(org.apache.iceberg.expressions.Literal<?> lit) {
     return null;
   }
 
+  private static List<String> getMultipartIdentifier(SparkSession spark, String name) throws ParseException {
```

Comment:
Nit: use of `get` in a method name signals either that there is a more descriptive verb (like "create") or that "get" can simply be removed (as in the case of a getter).

Reply:
Will change.